src/components/Editor/tests/Editor.spec.js

```diff
@@ -117,7 +117,6 @@ describe("Editor", () => {
       });
 
       expect(mockEditor.setText.mock.calls).toEqual([
-        [""],
```

Comment:
Is it okay that this one call is being skipped?

Reply:
yep